pom.xml

```diff
@@ -86,13 +86,13 @@
     </scm>
 
     <properties>
-        <revision>1.3.0</revision>
+        <revision>1.5.0.1</revision>
```

Comment:
@imbajin Which version number should we use here?

Reply:
> @imbajin Which version number should we use here?

`1.5.0` for master (we could change it if need)